src/components/NotificationModal.tsx

```diff
@@ -118,6 +109,25 @@ export default function NotificationModal({ location }: INotificationModalProps)
     hourCycle: 'h23'
   }
 
+  const getFilteredOptions = () => {
+    const now = DateTime.now();
+    const options = [
+      { value: "60", label: "1 hour before" },
+      { value: "180", label: "3 hours before" },
+      { value: "720", label: "12 hours before" },
+      { value: "1440", label: "24 hours before" },
+      { value: "2880", label: "48 hours before" },
+    ];
+
+    return options.filter(option => {
+      if (location.nextCleaningTime) {
+        const notificationTime = location.nextCleaningTime.minus({ minutes: parseInt(option.value) });
+        return notificationTime > now && notificationTime < location.nextCleaningTime;
```

Comment:
Can `notificationTime < location.nextCleaningTime`ever not be rue?

Reply:
good spot!